src/js/clinvar/eva-clinvar-widget.js

```diff
@@ -129,10 +129,10 @@ EvaClinVarWidget.prototype = {
             listeners: {
                 tabchange: function (tabPanel, newTab, oldTab, eOpts) {
                     _this.selectedToolDiv = newTab.contentEl;
-                    if (_this.lastVariant) {
```

Comment:
Where was this variable (and now `lastSelected`) declared? I can't find it, only the usages.